desktop/api/src/main/java/org/datacleaner/windows/AbstractWindow.java

```diff
@@ -49,18 +49,22 @@ public AbstractWindow(WindowContext windowContext) {
         setDefaultCloseOperation(JFrame.DO_NOTHING_ON_CLOSE);
         addWindowListener(this);
         getContentPane().setBackground(WidgetUtils.BG_COLOR_BRIGHT);
+
     }
 
+    protected abstract boolean isWindowMaximized();
```

Comment:
Instead of having to overwrite this in N places with "return false", why not just have a default implementation that returns false and then override it the one place where it is not?